Tests/test_main.py

```diff
@@ -0,0 +1,13 @@
+# import
```

Comment:
We don't need this file yet, so you can remove it for now. We will need something like this once we start to add functionality to the skeleton.